src/Util.js

```diff
@@ -13,22 +13,16 @@ function commonParams() {
     };
 }
 
-const TagRegex = /(refs\/)(tags\/M-staged-PR)(\d+)$/;
+const PrNumberRegex = / \(#(\d+)\)$/;
 
-function MatchTag(ref) {
-    return ref.match(TagRegex) !== null;
-}
-
-function ParseTag(ref) {
-    const matched = ref.match(TagRegex);
+function ParsePrNumber(prMessage) {
+    assert(prMessage);
+    const lines = prMessage.split(/\r*\n/);
+    const matched = lines[0].match(PrNumberRegex);
     if (!matched)
         return null;
-    return matched[3];
-}
-
-function StagingTag(prNum) {
-    assert(prNum);
-    return "tags/M-staged-PR" + prNum;
+    assert(matched[1] > 0);
```

Comment:
Let's name `matched[1]` prNumber or something like that.

Reply:
Done.